docs/src/main/docs/guides/index.md

```diff
@@ -10,6 +10,6 @@ permalink: guides
 These guides are aimed at developers who are already familiar with Mu-Scala.
 
 If you are new to Mu-Scala, we recommend you read the [Getting Started
-guide](../getting-started) and the [tutorials](../tutorials).
+guide](getting-started) and the [tutorials](tutorials).
```

Comment:
As the user is in the index.html, the path is omitted by the browser. It seems that it's not needed to go back with `..` .